h2/src/main/org/h2/table/Column.java

```diff
@@ -68,6 +68,7 @@
     private int displaySize;
     private Table table;
     private String name;
+    private int columnPos;
```

Comment:
Why do we need this additional thing `columnPos`. It looks really weird that we need it in addition to `columnId`.